source/_patterns/10-components/article-teaser/article-teaser.twig

```diff
@@ -9,4 +9,16 @@
       <p class="article-teaser__date">Mittwoch, 15. Februar 2017, 16:03 Uhr, aktualisiert um 17:57 Uhr</p>
     </div>
   </div>
+</a>
+
+<a href="#" class="article-teaser">
+  <span class="h-offscreen">Hinweis auf einen verwandten Artikel:</span>
+  <div class="article-teaser__wrapper">
+    <img src="{{ images[0].sources.s8.src }}" alt="Typ schweisst etwas" class="article-teaser__image" />
+
+    <div class="article-teaser__content">
+      <h4 class="article-teaser__kicker">Frankreich hat gewählt</h4>
```

Comment:
vielleicht könnte man "in diesem Rahmen" auch grad noch schnell die Titelreihenfolge anpassen?